### PR TITLE
feat[ch499]: clean up match details page

### PR DESCRIPTION
**Pull request by Jarrydcrawford**
- Use buttons and formatting to closely match the designs
- Show match time in-place of score if the played time is in the future

### CODE REVIEW

File: src/components/Versus/Versus.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { cleanup, render } from '@testing-library/react';
+import tk from 'timekeeper';
+import { Versus } from './Versus';
+
+describe('Versus', () => {
+  afterEach(() => {
+    cleanup();
```
Comment (harrygr) [inline src/components/Versus/Versus.spec.tsx]: We shouldn't need to do this in every test module if we just use the global RTL cleanup hook. Although I think the latest version now automatically cleans up after each test.

File: src/pages/Dashboard/Match/index.tsx [hunk 2]
```diff
@@ -73,6 +73,11 @@ export const DashboardMatchIndex = (props: Props) => {
                   path="/dashboard/m/:id/pick-squad"
                   exact={true}
                 />
+                <Route
+                  component={PickSquad}
+                  path="/dashboard/m/:id/view-squad"
+                  exact={true}
+                />
```
Comment (harrygr) [inline src/pages/Dashboard/Match/index.tsx]: I can't see amnywhere that links to this route and we already have `/pick-squad`. Do we need this?

File: src/pages/Match/MatchDetails.tsx [hunk 3]
```diff
@@ -30,60 +29,39 @@ export const MatchDetails: React.FunctionComponent<Props> = props => {
...
   }));
 
-  const canEditMatch =
-    props.match.votingStatus === 'PENDING' && props.isLoggedInAsOwner;
-
   return (
     <MatchDetailsContainer topBuffer={true}>
       <Vspace>
+        <PaddedCard space="lion">
+          {format(props.match.playedAt, 'ddd D MMM, HH:mm')}
```
Comment (harrygr) [inline src/pages/Match/MatchDetails.tsx]: This seems like repeated information as the date and time is above in the header
Screenshot 2019-08-20 at 09 14 51

File: src/pages/Match/OwnerMatchDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    <MatchDetailsContainer topBuffer={true}>
+      <Vspace>
+        <CardOrCardLink
+          actionLink={{
+            to: `/dashboard/m/${props.match.id}/edit`,
+            title: 'Edit match details',
+          }}
+          space="lion"
+        >
+          {format(props.match.playedAt, 'ddd D MMM, HH:mm')}
```
Comment (harrygr) [inline src/pages/Match/OwnerMatchDetails.tsx]: Again the date and time is above, maybe this card can just say something like "Edit match details"

File: src/pages/Dashboard/Match/index.tsx (hunk 2 above)
Comment (Jarrydcrawford) [reply src/pages/Dashboard/Match/index.tsx]: Woops left over code.

File: src/pages/Match/MatchDetails.tsx (hunk 3 above)
Comment (Jarrydcrawford) [reply src/pages/Match/MatchDetails.tsx]: Yeah good point, I'll update the wording for the owner and remove this for anon view.

**Pull request merged**